manifests/eap/client.pp

```diff
@@ -127,15 +150,16 @@
     }
 
     lightblue::client::configure{ "${module_path}/lightblue-client.properties":
-        owner                   => 'jboss',
-        group                   => 'jboss',
-        lbclient_metadata_uri   => $metadata_service_uri,
-        lbclient_data_uri       => $data_service_uri,
-        lbclient_use_cert_auth  => $use_cert_auth,
-        lbclient_ca_file_path   => $ssl_ca_file_path,
-        lbclient_cert_file_path => $auth_cert_file_path,
-        lbclient_cert_password  => $auth_cert_password,
-        lbclient_cert_alias     => $auth_cert_alias,
-        require                 => File[$module_dirs],
+        owner                       => 'jboss',
+        group                       => 'jboss',
+        lbclient_metadata_uri       => $metadata_service_uri,
+        lbclient_data_uri           => $data_service_uri,
+        lbclient_use_cert_auth      => $use_cert_auth,
+        lbclient_ca_file_path       => $ssl_ca_file_path,
```

Comment:
It doesn't need to be as it is no longer used, however I intentionally didn't remove it as this time.  I wanted to try and keep this as backwards compatible as possible for the time being since if we remove it now, client Puppet code will start breaking.  If there is a cleaner way to do versions and deprecation in Puppet, we could certainly go that direction.